Stratis.Bitcoin.Tests/Wallet/WalletManagerTest.cs

```diff
@@ -1330,24 +3185,62 @@ private Features.Wallet.Wallet GenerateBlankWallet(string name, string password)
                 AccountsRoot = new List<AccountRoot> { new AccountRoot { Accounts = new List<HdAccount>(), CoinType = (CoinType)Network.Main.Consensus.CoinType } },
             };
 
-            return walletFile;
+            return (walletFile, extendedKey);
         }
 
-        private ChainedBlock AppendBlock(ChainedBlock previous, params ConcurrentChain[] chains)
+        private Block AppendTransactionInNewBlockToChain(ConcurrentChain chain, Transaction transaction)
```

Comment:
I feel as though all of these methods can be made static and exported to a class that anyone can use for testing purposes. What do you think?